substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -785,6 +785,8 @@ pub mod pallet {
 		SnapshotTargetsSizeExceeded { size: u32 },
 		/// A new force era mode was set.
 		ForceEra { mode: Forcing },
+		/// A nominator has set their preferences.
+		NominatorPrefsSet { who: T::AccountId, prefs: Nominations<T> },
```

Comment:
I'm not necessarily opposed to this new event, however I'd rename it to `Nominated {who, nominations}` to be more clear and inline with the semantics we use in staking.